third_party/CMakeLists.txt

```diff
@@ -179,6 +179,8 @@ if(JPEGXL_ENABLE_TOOLS)
                      ${PROJECT_BINARY_DIR}/LICENSE.gflags COPYONLY)
     endif()  # JPEGXL_DEP_LICENSE_DIR
   endif()
+  # https://gitlab.kitware.com/cmake/cmake/-/issues/22919
+  set_target_properties(gflags PROPERTIES IMPORTED_GLOBAL TRUE)
```

Comment:
Getting cmake error "set_target_properties can not be used on an ALIAS target.".
Perhaps that can easily be fixed by move those two lines inside the ifs above?!